GoogleForCreators/web-stories-wp
# Shopping: Support custom CTA text

## Context



## Summary
 Allow for a custom call to action to be defined when a product is on canvas. 

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Enable shopping experiment. 
2. Active woocommerce and setup a test product. 
3. Create new story. 
4. Go to product tab, insert story. 
5. Click on page. 
6. Go to style tag. 
7. Change CTA text to something custom. 
8. See text change on button.  


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11309
